core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/console/CodeSystemsCommand.java

```diff
@@ -161,15 +244,43 @@ private CodeSystem getCodeSystemById(String shortNameOrOid) {
 				.getSync(1, TimeUnit.MINUTES);
 	}
 	
-	private String getCodeSystemVersionInformation(CodeSystemVersion codeSystemVersion) {
-		return new StringBuilder()
-			.append("\tVersion id: ").append(codeSystemVersion.getVersion()).append("\n")
-			.append("\tDescription: ").append(codeSystemVersion.getDescription()).append("\n")
-			.append("\tEffective date: ").append(EffectiveTimes.format(codeSystemVersion.getEffectiveTime(), DateFormats.DEFAULT)).append("\n")
-			.append("\tCreation date: ").append(Dates.formatByHostTimeZone(codeSystemVersion.getImportDate(), DateFormats.DEFAULT)).append("\n")
-			.append("\tLast update: ").append(codeSystemVersion.getLastModificationDate() != null ? StdDateFormat.getDateInstance().format(codeSystemVersion.getLastModificationDate()) : "-").append("\n")
-			.append("\tVersion branch path: ").append(codeSystemVersion.getPath())
-			.toString();
+	//I added the CodeSystem argument so i don't have to search for it in a request. It saves a lot of time
```

Comment:
I would either rewrite or remove this javadoc. Think about it in a way how someone would interpret this 5 years from now? This is more like a response for the PR review process rather than a useful message related to the functionality.

Reply:
Done, I removed the comment